packages/react-native-editor/jest_ui_test_environment.js

```diff
@@ -23,7 +23,7 @@ class CustomEnvironment extends JSDOMEnvironment {
 	}
 
 	async teardown() {
-		await this.global.editorPage.stopDriver();
+		await this.global.editorPage?.stopDriver();
```

Comment:
I'm not sure this is a good fix, mainly a hunch to see if it fixes the instability.

Reply:
`stopDriver` being undefined is usually the result of a failure with something else. But I understand that it causes confusion to see this error. I think it should be fine to have this change.